hazelcast-integration/kubernetes/hazelcast-enterprise.yaml

```diff
@@ -49,22 +49,8 @@ spec:
               name: hz-enterprise-license
               key: key
         - name: JAVA_OPTS
-          value: "-Dhazelcast.rest.enabled=true -Dhazelcast.config=/data/hazelcast/hazelcast.xml"
+          value: "-Dhazelcast.config=/data/hazelcast/hazelcast.yaml"
       volumes:
       - name: hazelcast-storage
         configMap:
           name: hazelcast-configuration
-
----
-
-apiVersion: v1
```

Comment:
so without a k8s service, it will discover all pods in the namespace?

Reply:
yes, I think for such simple scenario it's good enough.